### PR TITLE
Unflag page

### PR DESCRIPTION
**Pull request by 0xjocke**
Add unflag page.
Also add links to mint, burn and unflag from main action cards
image

### CODE REVIEW

File: v2/components/UnflagOptions/UnflagOptions.tsx [hunk 1]
```diff
@@ -136,19 +150,38 @@ export const UnflagOptionsUi: React.FC<{
...
+        <Button
+          variant="outline"
+          disabled={!continueLink}
+          _disabled={{ color: 'gray.600', borderColor: 'gray.600', cursor: 'not-allowed' }}
+          _hover={{ bg: 'none' }}
+          onClick={() => {
+            if (continueLink) {
+              navigate(continueLink);
+            }
+          }}
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/UnflagOptions/UnflagOptions.tsx:167]: `navigate` type only accept a string and typescript isn't smart enough to figure out that the `disabled` prop disabled `onClick`. So this if is just to make typescript happy

File: v2/cypress/cypress/support/component.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { mount } from 'cypress/react18';
+import { BrowserRouter } from 'react-router-dom';
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/cypress/cypress/support/component.js:2]: Yep that make sense. I'm not actually testing any routes just making sure useNavigate doesn't throw missing context. But will update to memory router either way

File: v2/cypress/package.json [hunk 3]
```diff
@@ -49,5 +49,8 @@
       "buffer",
       "process"
     ]
+  },
+  "dependencies": {
+    "react-router-dom": "^6.3.0"
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/cypress/package.json:54]: Will fix

File: v2/components/UnflagOptions/UnflagOptions.tsx [hunk 4]
```diff
@@ -8,12 +8,18 @@ import {
...
   Skeleton,
+  Divider,
+  Link,
+  Button,
 } from '@chakra-ui/react';
 import { CollectIcon, InfoIcon, MaintainIcon } from '@snx-v2/icons';
 import { useDebtData } from '@snx-v2/useDebtData';
 import { useTranslation } from 'react-i18next';
 import { useLiquidationData } from '@snx-v2/useLiquidationData';
 import { formatPercent } from '../../ui/utils/formatters/number';
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/UnflagOptions/UnflagOptions.tsx:19]: Thanks!

File: v2/components/MainActionCards/MainActionCards.tsx [hunk 5]
```diff
@@ -67,7 +70,7 @@ const StakeActionCard: React.FC<UiProps> = ({ currentCRatioPercentage }) => {
       {isStaking ? (
         <Button
           onClick={() => {
-            console.log('navigate to stake and borrow more');
+            navigate('/staking/mint');
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/MainActionCards/MainActionCards.tsx:73]: Yea maybe it's a little less error prone to have it in one place. Currently we dont have a package for it. You have opinions about this @jmzwar ?

File: v2/components/MainActionCards/MainActionCards.tsx [hunk 6]
```diff
@@ -262,6 +277,7 @@ export const MainActionCards: React.FC = () => {
   const { data: feePoolData } = useFeePoolData();
   const { data: rewardsData } = useRewardsAvailable();
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/MainActionCards/MainActionCards.tsx:278]: This is just an example for doing mutative actions (for you :)  hehe) . I'm removing it in this PR https://github.com/Synthetixio/js-monorepo/pull/360

**Review by noisekit — APPROVED** [review]

**Pull request merged**